CHANGELOG.rst

```diff
@@ -1,3 +1,20 @@
+4.3.0 (2023-09-20)
+------------------
```

Comment:
We probably won't have an immediate release, but even so the process is to set version/date at the time of release. 👍 

```suggestion
UNRELEASED
----------
```